src/components/Layouts/Primary/TopBar/Items/NavMenu/NavMenu.tsx

```diff
@@ -63,11 +86,123 @@ const NavMenu = (): ReactElement => {
+                        autoFocusItem={reportsMenuOpen}
+                        id="menu-list-grow"
+                      >
+                        <NextLink
+                          href="/accountLists/[accountListId]/reports/donations"
+                          as={`/accountLists/${state.accountListId}/reports/donations`}
+                          scroll={false}
+                        >
+                          <MenuItem onClick={handleReportsMenuClose}>
+                            <ListItemText primary={t('Donations')} />
+                          </MenuItem>
+                        </NextLink>
+                        <NextLink
+                          href="/accountLists/[accountListId]/reports/partner"
+                          as={`/accountLists/${state.accountListId}/reports/partner`}
+                          scroll={false}
+                        >
+                          <MenuItem onClick={handleReportsMenuClose}>
+                            <ListItemText
+                              primary={t('14-Month Report (Partner)')}
+                            />
+                          </MenuItem>
+                        </NextLink>
+                        <NextLink
+                          href="/accountLists/[accountListId]/reports/salary"
+                          as={`/accountLists/${state.accountListId}/reports/salary`}
+                          scroll={false}
+                        >
+                          <MenuItem onClick={handleReportsMenuClose}>
+                            <ListItemText
+                              primary={t('14-Month Report (Salary)')}
+                            />
+                          </MenuItem>
+                        </NextLink>
+                        <NextLink
+                          href="/accountLists/[accountListId]/reports/designation_accounts"
+                          as={`/accountLists/${state.accountListId}/reports/designation_accounts`}
+                          scroll={false}
+                        >
+                          <MenuItem onClick={handleReportsMenuClose}>
+                            <ListItemText primary={t('Designation Accounts')} />
+                          </MenuItem>
+                        </NextLink>
+                        <NextLink
+                          href="/accountLists/[accountListId]/reports/financial_accounts"
+                          as={`/accountLists/${state.accountListId}/reports/financial_accounts`}
+                          scroll={false}
+                        >
+                          <MenuItem onClick={handleReportsMenuClose}>
+                            <ListItemText
+                              primary={t('Responsibility Centers')}
+                            />
+                          </MenuItem>
+                        </NextLink>
+                        <NextLink
+                          href="/accountLists/[accountListId]/reports/monthly"
```

Comment:
I think you could make a few of these report urls more descriptive. A "monthly" report doesn't mean much since there are several that show data over the months.